### ISSUE TITLE
'migrations' table was not updated

### ISSUE THREAD

**Issue by dordana** [Xethron/migrations-generator#202]
After I ran `php artisan migrate:generate` 
After running the command `php artisan migrate:generate`` the process created files for me but did not update the 'migrations' table in DB.

**Comment by jasperf** [comment]
That is true. It does not update the migrations table and so on next migration it will complain tables already exist as it seemingly never ran those migrations before. 

In our case we replaced old migrations as after many new ones they did not work for new setups anymore. But now we can only run single migration files. 

Genera migrations hang on table already exist and we do not want to overwrite, but we just want to skip. Not sure if there are solutions for that though.
